### PR TITLE
Add tests for `RandomBeaconHistory` contract

### PR DESCRIPTION
**Pull request by m-Peter**
Work towards https://github.com/onflow/developer-grants/issues/216

# Description

Uses the Cadence testing framework to test the functionality of `RandomBeaconHistory` contract.

### CODE REVIEW

File: transactions/randomBeaconHistory/scripts/get_source_of_randomness_page.cdc [hunk 1]
```diff
@@ -3,5 +3,5 @@ import "RandomBeaconHistory"
 /// Retrieves the source of randomness for the requested block height from the RandomBeaconHistory contract.
 ///
 access(all) fun main(page: UInt64, perPage: UInt64): RandomBeaconHistory.RandomSourceHistoryPage {
-    return RandomBeaconHistory.getRandomSourceHistoryPage(page: page, perPage: perPage)
+    return RandomBeaconHistory.getRandomSourceHistoryPage(page, perPage: perPage)
```
Comment (m-Peter) [inline transactions/randomBeaconHistory/scripts/get_source_of_randomness_page.cdc:6]: cc @janezpodhostnik There was an error while testing this script, not how much important it is though.

**Review by joshuahannan — APPROVED** [review]

File: transactions/randomBeaconHistory/scripts/get_source_of_randomness_page.cdc (hunk 1 above)
Comment (janezpodhostnik) [reply transactions/randomBeaconHistory/scripts/get_source_of_randomness_page.cdc:6]: Its not that important, as it is mostly as an example, but we should fix this. I'll open up a PR.

Comment (m-Peter) [reply transactions/randomBeaconHistory/scripts/get_source_of_randomness_page.cdc:6]: It is fixed in this PR, so not sure there's a need to open up another PR.

File: flow.json [hunk 2]
```diff
@@ -1,30 +1,106 @@
...
+    "RandomBeaconHistory": {
+      "source": "./contracts/RandomBeaconHistory.cdc",
+      "aliases": {
+        "testing": "0x0000000000000007"
+      }
+    },
+    "StakingProxy": {
+      "source": "./contracts/StakingProxy.cdc",
+      "aliases": {
+        "testing": "0x0000000000000007"
```
[the remark this answers is not in the document]
Comment (m-Peter) [reply flow.json:90]: The `testing` alias is basically the account where the current contract will be deployed for usage in the Cadence testing framework. For the sake of completion & consistency, I had added it to all contracts in the `flow.json` config file. It does not need to be the same though. There are 10 predefined accounts that can be used as the testing alias.

**Comment by m-Peter** [comment]
The CI does not automatically run for pull requests coming from forks.

**Pull request merged**